internal/pkg/install/mount.go

```diff
@@ -18,17 +16,13 @@ import (
 // to the appropriate mountpoint.
 // TODO: See if we can consolidate this with rootfs/mount
 func Mount() (err error) {
-	log.Println("Discovering mountpoints")
```

Comment:
Simplify logging a bit to avoid kernel print rate limiting. We can add it back later when we have level logging, and can disable kernel print rate limiting based on the log level.